### ISSUE TITLE
Where to generate Pull Request for new template

### ISSUE THREAD

**Issue by nikhilranosys** [OpenCerts/opencerts-website#390]
Hi,

I have developed my custom template bu when I started that time templates were in https://github.com/OpenCerts/opencerts-website and now as I can see they are moved to https://github.com/OpenCerts/legacy-templates.

So shall I fork legacy-template repo and generate pull request there or pull request will still be generated on opencerts-website?

**Comment by rjchow** [comment]
Is this for an SSG accredited institution?

**Comment by nikhilranosys** [comment]
yes, it is for Singapore Management University.

**Comment by rjchow** [comment]
in that case do it against the legacy-template repo - it should work the same

**Comment by nikhilranosys** [comment]
Where pull request will go for institutions without SSG accredition.

Actually we are also planing to offer OpenCerts implementation as solution to our other customers (universities in singapore) also so want to know.

**Comment by nikhilranosys** [comment]
I have forked and cloned the legacy-template repo to add my custom template but  after running `npm install` and `npm run dev` commands when I open http://localhost:3000 it just shows a blank page. I am yet to add my custom templates. But because I can't view the certificate viewer how I can test my template after adding it to the legacy-template base?

I want to mention here that when I started working on my custom templates I forked opencerts-viewer repo on 2nd July 2019.
